src/app/pages/footer/footer.component.ts

```diff
@@ -5,7 +5,6 @@ import {
 
```

Comment:
Dopiero teraz to zauważyłem... ale footer i header raczej nie powinien być w folderze `pages`. Z tego co rozumiem, `pages` ma za zadanie odzwierciedlać logikę stron jakie można spotkać w aplikacji (home, courseList, addCourse, login itp), a to są komponentu dzielone na wszystkich stronach, także jakbyś dodał ten moduł `sharedModule`, o którym wspomniałem wcześniej mógłbyś te komponenty przenieść tam ;)